File: Swift/Lista_04_EX04/UsuarioRegular.swift
import Foundation
 
 public class UsuarioRegular : Usuario{
 
	override func descricaoRegular(){
		print("O usuario regular pode ler arquivos e auterar arquivos. ")
		print("Area de usuário cadastrado")
	}
	
	func despedidaRegular( ) {
		print("Até logo \(login)!")
	}
	
}

File: Java/Lista 4/L4_Ex1.4/src/l4_ex1/pkg4/UsuarioRegular.java

package l4_ex1.pkg4;

public class UsuarioRegular extends Usuario{
    
    public UsuarioRegular(String login) {
        super(login);
    }
    
    @Override
    public void descricao(){
        System.out.println("O usuario regular pode ler arquivos e auterar arquivos");
        System.out.println("Area de usuário cadastrado");
    }
    
    public void despedidaRegular(){
        System.out.println("Até logo " + this.getLogin() + "!");
    }
    
}
